docs/quickstart.md

```diff
@@ -4,20 +4,28 @@ Sedna is an open source framework for edge-cloud collaborative training and infe
 
 ### Get Sedna
 
-You can find the latest Sedna release [here](TODO)
+You can find the latest Sedna release [here](https://github.com/kubeedge/sedna/releases)
```

Comment:
```suggestion
You can find the latest Sedna release [here](https://github.com/kubeedge/sedna/releases)
```
Sedna releases are now listed on the [releases](https://github.com/kubeedge/sedna/releases) page.